### DISCUSSION TITLE
Usage of clr in IronPython

### DISCUSSION THREAD

**Discussion by AkkSamuel** [IronLanguages/ironpython3#1691] [Q&A]
Hi, I'm new to using IronPython (on Spotfire) and I have to reverse engineer some code, I'm just getting one issue about the usage of clr. I'm not sure I understand what this module is for, and I have troubles with functions like AddReference, which I really just don't understand, and I can't get a satisfying explanation on why and how it is being used on spotfire. I'm not sure that I'm asking it at the right place, I'm also new to github so please tell me if I should ask somewhere else. Thank you !

**Comment by BCSharp** [comment]
Look at [IronPython documentation](https://ironpython.net/documentation/dotnet/). The current documentation page covers only up to v2.7, but the basics like _clr_ are mostly the same for v3.4.
